### ISSUE TITLE
DPO训练log打印日志:logits/chosen和logits/rejected完全一样

### ISSUE THREAD

**Issue by qq941134965** [modelscope/ms-swift#3880]
其它指标都是正常进行的，只有logits/chosen和logits/rejected这两个指标完全一样，这是为什么：
{'loss': 0.51571407, 'grad_norm': 29.52262878, 'learning_rate': 6.9e-07, 'memory(GiB)': 65.72, 'train_speed(iter/s)': 0.082387, 'rewards/chosen': 0.38325197, 'rewards/rejected': -1.1050781, 'rewards/accuracies': 0.89375001, 'rewards/margins': 1.48847651, 'logps/chosen': -0.88531494, 'logps/rejected': -6.45234394, 'logits/chosen': -2.5406251, 'logits/rejected': -2.5406251, 'nll_loss': 0.19442749, 'epoch': 0.82, 'global_step/max_steps': '805/1966', 'percentage': '40.95%', 'elapsed_time': '2h 42m 16s', 'remaining_time': '3h 54m 2s'}
{'loss': 0.58782959, 'grad_norm': 33.83953476, 'learning_rate': 6.8e-07, 'memory(GiB)': 65.72, 'train_speed(iter/s)': 0.082502, 'rewards/chosen': 0.38481444, 'rewards/rejected': -0.92888182, 'rewards/accuracies': 0.82499999, 'rewards/margins': 1.31308591, 'logps/chosen': -1.01716304, 'logps/rejected': -5.84687519, 'logits/chosen': -2.54375005, 'logits/rejected': -2.54453135, 'nll_loss': 0.22470704, 'epoch': 0.82, 'global_step/max_steps': '810/1966', 'percentage': '41.20%', 'elapsed_time': '2h 43m 3s', 'remaining_time': '3h 52m 43s'}
{'loss': 0.51054478, 'grad_norm': 27.50166512, 'learning_rate': 6.8e-07, 'memory(GiB)': 65.72, 'train_speed(iter/s)': 0.08262, 'rewards/chosen': 0.4144043, 'rewards/rejected': -0.94926757, 'rewards/accuracies': 0.85000002, 'rewards/margins': 1.36347651, 'logps/chosen': -0.72973633, 'logps/rejected': -6.01171875, 'logits/chosen': -2.5460937, 'logits/rejected': -2.5476563, 'nll_loss': 0.16516724, 'epoch': 0.83, 'global_step/max_steps': '815/1966', 'percentage': '41.45%', 'elapsed_time': '2h 43m 50s', 'remaining_time': '3h 51m 23s'}
{'loss': 0.49804497, 'grad_norm': 35.20991898, 'learning_rate': 6.8e-07, 'memory(GiB)': 65.72, 'train_speed(iter/s)': 0.08273, 'rewards/chosen': 0.46889648, 'rewards/rejected': -0.92998046, 'rewards/accuracies': 0.88749999, 'rewards/margins': 1.3992188, 'logps/chosen': -0.62480468, 'logps/rejected': -5.91796875, 'logits/chosen': -2.55703115, 'logits/rejected': -2.55703115, 'nll_loss': 0.1571289, 'epoch': 0.83, 'global_step/max_steps': '820/1966', 'percentage': '41.71%', 'elapsed_time': '2h 44m 37s', 'remaining_time': '3h 50m 4s'}
{'loss': 0.48759613, 'grad_norm': 34.94580841, 'learning_rate': 6.7e-07, 'memory(GiB)': 65.72, 'train_speed(iter/s)': 0.082839, 'rewards/chosen': 0.43613282, 'rewards/rejected': -1.09179688, 'rewards/accuracies': 0.90625, 'rewards/margins': 1.52792966, 'logps/chosen': -0.64998782, 'logps/rejected': -6.46640635, 'logits/chosen': -2.5484376, 'logits/rejected': -2.5476563, 'nll_loss': 0.17774048, 'epoch': 0.84, 'global_step/max_steps': '825/1966', 'percentage': '41.96%', 'elapsed_time': '2h 45m 25s', 'remaining_time': '3h 48m 46s'}

**Comment by Jintao-Huang** [comment]
main分支修复了。

这个指标对训练loss没有影响的
